### ISSUE TITLE
DomainNet train-test splits

### ISSUE THREAD

**Issue by tarun005** [JinjingZhu/PMTrans#17]
Did you use the complete train+test .txt files from DomainNet for training, because in dataset/domainnet/ I only find one file for each domain and no train-test splits? This is important because prior works mostly adopt a train-test split using the respective files.

**Comment by JinjingZhu** [comment]
Maybe you make some misunderstanding about the training and test procedure of UDA. Please find some basic codes for knowing well. Like DSAN, https://github.com/jindongwang/transferlearning/tree/master. Best wishes!

**Comment by tarun005** [comment]
I disagree. The norm for using DomainNet has been to use separate train-test splits, as done in 

1. https://github.com/ViLab-UCSD/MemSAC_ECCV2022
2. https://github.com/microsoft/UDA/blob/main/datasets/domainnet.py

Don't you think training using test-set is wrong?

**Comment by SuperBigDevil** [comment]
域适应不就应该通过target 的分类正确率来判定方法有效性吗

**Comment by JinjingZhu** [comment]
如果方便，可以加我微信进行讨论：naivezhu

祝好！
